Add list of supported browsers to README

gordlin opened the pull request:
### Related Item(s)
Issue #1891 

### Changes
- [DOCS] Add a list of supported/semi-supported browsers to README.md.

### Notes
The browsers have been categorized as so:
**Full support**: Chromium browsers (Chrome, Edge), Firefox
**Limited support**: Opera, Safari, IE

If you'd like to have Netscape mentioned, please let me know.

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)
<!-- Reviewable:end -->

james-rae reviewed (changes requested):
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 2 unresolved discussions (waiting on @gordlin)

___
*[`README.md` line 18 at r1](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz4iNkx7PI8YDNEWvOP:-Nz4iNkx7PI8YDNEWvOQ:b-qg5sjo) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/1704e6b5e8f8c396c446e7947271a3d83127a199/README.md#L18)):*
> ```Markdown
> ![](https://byob.yarr.is/ramp4-pcar4/ramp4-pcar4/tsbadge) ![](https://byob.yarr.is/ramp4-pcar4/ramp4-pcar4/lintbadge)
> 
> ## Browser Support
> ```

Looks good, but I'd move to below the `Documentation` section. Odds are a visitor will be more interested in the docs. Everyone loves reading docs.
___
*[`README.md` line 20 at r1](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz4idWs1kBqVyHd2lIh:-Nz4idWs1kBqVyHd2lIi:b-8k5i8q) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/1704e6b5e8f8c396c446e7947271a3d83127a199/README.md#L20)):*
> ```Markdown
> ## Browser Support
> 
> There is **full support** for modern versions of Chromium-based browsers (Google Chrome, Edge, etc.) and Firefox. Keep your browser up-to-date to ensure full functionality.
> ```

suggested word change: `Keep your` --> `We suggest keeping your`.  Sounds less commanding :person_with_crown: 


<!-- Sent from Reviewable.io -->

gordlin commented:
> > Reviewed 1 of 1 files at r1, all commit messages.
> > _[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)_ status: all files reviewed, 2 unresolved discussions (waiting on @gordlin)
> 
> _[`README.md` line 18 at r1](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz4iNkx7PI8YDNEWvOP:-Nz4iNkx7PI8YDNEWvOQ:b-qg5sjo) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/1704e6b5e8f8c396c446e7947271a3d83127a199/README.md#L18)):_
> 
> > ```
> > ![](https://byob.yarr.is/ramp4-pcar4/ramp4-pcar4/tsbadge) ![](https://byob.yarr.is/ramp4-pcar4/ramp4-pcar4/lintbadge)
> > 
> > ## Browser Support
> > ```
> 
> Looks good, but I'd move to below the `Documentation` section. Odds are a visitor will be more interested in the docs. Everyone loves reading docs.
> 
> _[`README.md` line 20 at r1](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz4idWs1kBqVyHd2lIh:-Nz4idWs1kBqVyHd2lIi:b-8k5i8q) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/1704e6b5e8f8c396c446e7947271a3d83127a199/README.md#L20)):_
> 
> > ```
> > ## Browser Support
> > 
> > There is **full support** for modern versions of Chromium-based browsers (Google Chrome, Edge, etc.) and Firefox. Keep your browser up-to-date to ensure full functionality.
> > ```
> 
> suggested word change: `Keep your` --> `We suggest keeping your`. Sounds less commanding 🫅

Donethanks.

james-rae reviewed (changes requested):
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @gordlin)

___
*[`README.md` line 28 at r2](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz8TxtP3R9EdG2Sanuf:-Nz8TxtP3R9EdG2Sanug:be1blrg) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/99edd50ea543d26f7bd06a340a01571caac8b732/README.md#L28)):*
I guess I glossed over this yesterday.  For these three, it is true that we are not testing them as hardcore as the full-supported list. However, Opera is chromium (or so the internet tells me), so I think this falls under the `etc` above. Safari we do test from time to time...has a bit of sketch but I don't think we should be "not recommending" it. IE however, you are bang on, not recommended.

So keep the last line but just make it about IE. Remove Opera alltogether?? Put a line in the middle for Safari, maybe something like

> Safari works well but may experience some sub-optimal UI behavior. Please log any issues encountered.

Don't love the text but best I got.

If you did any research for this list and I'm telling big lies here, please retort.


<!-- Sent from Reviewable.io -->

gordlin commented:
> > Reviewed 1 of 1 files at r2, all commit messages.
> > _[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)_ status: all files reviewed, 1 unresolved discussion (waiting on @gordlin)
> 
> _[`README.md` line 28 at r2](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz8TxtP3R9EdG2Sanuf:-Nz8TxtP3R9EdG2Sanug:be1blrg) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/99edd50ea543d26f7bd06a340a01571caac8b732/README.md#L28)):_ I guess I glossed over this yesterday. For these three, it is true that we are not testing them as hardcore as the full-supported list. However, Opera is chromium (or so the internet tells me), so I think this falls under the `etc` above. Safari we do test from time to time...has a bit of sketch but I don't think we should be "not recommending" it. IE however, you are bang on, not recommended.
> 
> So keep the last line but just make it about IE. Remove Opera alltogether?? Put a line in the middle for Safari, maybe something like
> 
> > Safari works well but may experience some sub-optimal UI behavior. Please log any issues encountered.
> 
> Don't love the text but best I got.
> 
> If you did any research for this list and I'm telling big lies here, please retort.

I've moved Opera to the 'Chromium-Based Browsers' section (probably safe to assume it works perfectly, given the other Chromium browsers do), added your line about Safari, and limited the 'limited support' to IE.

milespetrov reviewed (approved):
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 2 unresolved discussions (waiting on @gordlin and @james-rae)

___
*[`README.md` line 28 at r2](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz8TxtP3R9EdG2Sanuf:-NzDoCiU-CZ6wO6BElz0:b-b0gjqn) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/99edd50ea543d26f7bd06a340a01571caac8b732/README.md#L28)):*
<details><summary><i>Previously, gordlin (Gordon Lin) wrote…</i></summary><blockquote>

I've moved Opera to the 'Chromium-Based Browsers' section (probably safe to assume it works perfectly, given the other Chromium browsers do), added your line about Safari, and limited the 'limited support' to IE.
</blockquote></details>

A agree Opera should be removed if we're not testing against it. Chromium based browsers are largely similar but they do not support all the same features.
___
*[`README.md` line 30 at r3](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-NzDouOO8fuQOGc1ZrXW:-NzDouOO8fuQOGc1ZrXX:bs3co5v) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/ab7741166c0f4ea094bf35ca4857e973a8020cc7/README.md#L30)):*
> ```Markdown
> Safari works well but may experience some sub-optimal UI behaviour. Please log any issues encountered.
> 
> Internet Explorer has **limited support** and is not recommended.
> ```

I'd be surprised if ramp works in IE at all. Maybe go with not supported?


<!-- Sent from Reviewable.io -->

james-rae commented:
*[`README.md` line 28 at r2](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz8TxtP3R9EdG2Sanuf:-NzEb9pK82j6CDTsn9VU:b-zdt9b7) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/99edd50ea543d26f7bd06a340a01571caac8b732/README.md#L28)):*
<details><summary><i>Previously, milespetrov (Miles Petrov) wrote…</i></summary><blockquote>

A agree Opera should be removed if we're not testing against it. Chromium based browsers are largely similar but they do not support all the same features.
</blockquote></details>

Maybe we slap `Meeting Topic` on this one and haggle out what we actually want to say. Might need some tests against various browsers.


<!-- Sent from Reviewable.io -->

gordlin reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @james-rae and @milespetrov)

___
*[`README.md` line 28 at r2](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nz8TxtP3R9EdG2Sanuf:-Nz_-u537XgkF0eHVBNA:b-ktj5k) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/99edd50ea543d26f7bd06a340a01571caac8b732/README.md#L28)):*
<details><summary><i>Previously, james-rae (James Rae) wrote…</i></summary><blockquote>

Maybe we slap `Meeting Topic` on this one and haggle out what we actually want to say. Might need some tests against various browsers.
</blockquote></details>

I've modified the text based on the feedback so far.
___
*[`README.md` line 30 at r3](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-NzDouOO8fuQOGc1ZrXW:-Nz_-xvk0NLfBzubd2LI:bah2568) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/ab7741166c0f4ea094bf35ca4857e973a8020cc7/README.md#L30)):*
<details><summary><i>Previously, milespetrov (Miles Petrov) wrote…</i></summary><blockquote>

I'd be surprised if ramp works in IE at all. Maybe go with not supported?
</blockquote></details>

Donethanks


<!-- Sent from Reviewable.io -->

james-rae reviewed (approved):
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @milespetrov)



<!-- Sent from Reviewable.io -->

james-rae commented:
*[`README.md` line 28 at r4](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nzd-rQe0f3vaSnpzSoR:-Nzd-rQe0f3vaSnpzSoS:bs978u1) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/6e507d2cd73c3206f382769a22a88a23dc8a8146/README.md#L28)):*
> ```Markdown
> RAMP **supports** modern versions of Google Chrome, Microsoft Edge, and Firefox on desktop, and Google Chrome, Firefox, and Safari (iOS) on mobile devices. We suggest keeping your browser up-to-date to ensure full functionality.
> 
> Internet Explorer is **not supported**. [Use at your own discretion](https://i.imgur.com/zsh7Vkr.jpeg). 
> ```

You can use this sneaky url to get same effect without needing to send viewer to a site with ads

```
https://github.com/ramp4-pcar4/ramp4-pcar4/assets/7585245/4ac2deb1-3ace-4950-a687-6f9588f347ab
```


<!-- Sent from Reviewable.io -->

gordlin reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @james-rae and @milespetrov)

___
*[`README.md` line 28 at r4](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205#-Nzd-rQe0f3vaSnpzSoR:-NzdbIPz-YbK-per9qNa:b-ugpthr) ([raw file](https://github.com/ramp4-pcar4/ramp4-pcar4/blob/6e507d2cd73c3206f382769a22a88a23dc8a8146/README.md#L28)):*
<details><summary><i>Previously, james-rae (James Rae) wrote…</i></summary><blockquote>

You can use this sneaky url to get same effect without needing to send viewer to a site with ads

```
https://github.com/ramp4-pcar4/ramp4-pcar4/assets/7585245/4ac2deb1-3ace-4950-a687-6f9588f347ab
```
</blockquote></details>

Donethanks!


<!-- Sent from Reviewable.io -->

milespetrov reviewed (approved):
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @james-rae)



<!-- Sent from Reviewable.io -->

IshavSohal reviewed (approved):
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @james-rae)



<!-- Sent from Reviewable.io -->

sharvenp reviewed (approved):
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @james-rae)



<!-- Sent from Reviewable.io -->

yileifeng reviewed (approved):
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @james-rae)



<!-- Sent from Reviewable.io -->

dnlnashed reviewed (approved):
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @james-rae)



<!-- Sent from Reviewable.io -->

szczz reviewed (approved):
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @james-rae)



<!-- Sent from Reviewable.io -->

szczz reviewed (approved):
> Reviewed 1 of 1 files at r5.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: all files reviewed, 1 unresolved discussion (waiting on @james-rae)



<!-- Sent from Reviewable.io -->

james-rae reviewed (approved):
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @gordlin)



<!-- Sent from Reviewable.io -->

szczz reviewed (approved):
> Reviewed 1 of 1 files at r6, all commit messages.
> Dismissed @james-rae from a discussion.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2205)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @gordlin)



<!-- Sent from Reviewable.io -->
